shell/platform/fuchsia/dart_runner/tests/fidl/echo.fidl

```diff
@@ -1,8 +1,8 @@
-// Copyright 2019 The Fuchsia Authors. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
2023? Should this change be undone?

Reply:
All of the files in this repo use the flutter license, which for some reason starts with "Copyright 2013" :shrug: 

I just did this to make the license checker happy